src/plone/restapi/tests/test_serializer_summary.py

```diff
@@ -252,4 +252,5 @@ def test_dx_event_with_recurrence(self):
         ocurrence = ot.publishTraverse(self.request, tomorrow_str)
         summary = getMultiAdapter((ocurrence, self.request), ISerializeToJsonSummary)()
 
-        self.assertEqual(summary, {})
+        self.assertEqual(summary["start"], tomorrow_str)
+        self.assertEqual(summary["Title"], ocurrence.Title())
```

Comment:
plone.restapi also supports older versions of Plone that do not have the change from plone.app.event. Could you make this test conditional on the plone.app.event version that is installed?